hub/templates/hub/area/_mp_data.html

```diff
@@ -109,6 +109,38 @@ <h6 class="text-muted fw-bold">APPGs</h6>
                 <p class="card-text fs-8"><a href="#" class="text-decoration-none text-muted">Data collated by The Climate Coalition</a></p>
             </div>
         </div>
+  <div class="card flex-grow-1 dataset-card">
+             <div class="dataset-card-header">
+              <h5>Relevant votes & amendments</h5>
+             </div>
+             <div class="card-body">
+                <table class="table mb-0">
+                <thead>
+                    <tr>
+                        <th scope="col">Vote</th>
+                        <th scope="col">Position</th>
+                    </tr>
+                </thead>
+                <tbody>
+                  {% for vote in mp.votes %}
+                    <tr>
+                        <th><a href="{{ vote.url }}">{{ vote.name }}</a></th>
```

Comment:
My only change in this PR would be to use `<td>`s instead of `<th>`s around these vote and amendment names. They’re quite long titles, made even longer when bolded, and with only two cells per row, it feels like the "row header" styling hinders more than it helps.